docs/docs/launch-manual/sections/core-hacking/help.md

```diff
@@ -0,0 +1,9 @@
+## Having trouble?
+
+If you have any issues then please feel free to ask for help from the Pulsar Team:
+- [Discord server](https://discord.gg/gp9UStzsHk)
+- [subreddit](https://www.reddit.com/r/pulsaredit/)
+- [GitHub Discussions](https://github.com/pulsar-edit/pulsar/discussions)
+
+If you think you have found a bug in the process then please create a
+[new issue](https://github.com/pulsar-edit/pulsar/issues).
```

Comment:
Let's move this to a general FAQ section. This is the only other change I'd make. Then we could include specific lines as needed in these sections, but have it generally accessible from a main link.

```
@include(filename{start-end})
```
(from the documentation on this, we'll have to make sure to update the readme with this kind of information as well. But I think this will make things a bit simpler especially once something like this is.

There's probably even some other configs we can add since I added the theme last night:
https://vuepress-theme-hope.github.io/v2/guide/markdown/include.html#advanced

Reply:
See new latest commits and comments